src/cubi_tk/sodar/ingest_data.py

```diff
@@ -338,32 +337,17 @@ def _check_col_index_and_get_val(self, column_index, sheet_type, sheet_header, s
     def find_collection_name(self, sample_name, column_match, m):
         if column_match is None:
             return sample_name
-        val = column_match[sample_name]
+        val = column_match[str(sample_name)]
+        logger.debug(val)
         if not isinstance(val, list):
             return val
-        #needs further matching with conservation method and if existent tumor
-        conservation_col_name = None
-        conservation = m.groupdict(default=None)["conservation"]
+        #needs further matching with tumor
         tissue = m.groupdict(default=None)["tissue"]
```

Comment:
Please add a comment to indicate when the initial return is supposed to happen and/or just explicitly return/throw an error before the onko specific tumor checks happen.